webrender/src/picture.rs

```diff
@@ -118,11 +122,15 @@ pub struct Tile {
     /// cache handle can be used. Tiles are invalidated during the
     /// build_dirty_regions method.
     is_valid: bool,
+    /// The tile id is stable between display lists and / or frames,
+    /// if the tile is retained. Useful for debugging tile evictions.
+    id: TileId,
```

Comment:
I assume there are few enough tiles in the system and they're moved infrequently enough that an extra word for debugging has negligible overhead?

Reply:
Yea, typically 10 - 20 tiles ever active.